### DISCUSSION TITLE
convert (to pdf) using -annotate and %p : starting page should be 1 !

### DISCUSSION THREAD

**Discussion by schwedenpaule** [ImageMagick/ImageMagick#3272] [Help] [answered]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am using the latest version of ImageMagick
- [x] I have searched [open](https://github.com/ImageMagick/ImageMagick/issues) and [closed](https://github.com/ImageMagick/ImageMagick/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
it seems its not possible to print the page numbers starting with 1 (instead of 0), or am I missing something ?
"-frame 1" is not working here - I tried for hours and hours 

### Steps to Reproduce

convert i1.jpg i2.jpg i3.jpg -annotate +0+0 "page %p from %n" "result.pdf"
(result : the label on the first pdf-Page reads "page 0 from 3", but I'd like to achieve "page 1 from 3")

### System Configuration

- ImageMagick version: latest
- Environment (Operating system, version and so on): Windows 7
- Additional information:

**Comment by urban-warrior** [comment, marked as answer]
Try this:

```
convert i1.jpg i2.jpg i3.jpg -annotate +0+0 "page %[fx:t+1] from %n" "result.pdf"
```

**Comment by schwedenpaule** [comment]
(whow, this really does the trick! - thanks a lot!
(maybe the documentation (near the '%p'-Thing, and the 'fx:t'-Thing) should give some clear 'hints'))
